docs/contributing-to-registry.md

```diff
@@ -0,0 +1,66 @@
+>**Note**: Please ignore the doc for now as it's a work in progress. We are still working on creating the Bicep public registry, and the tool mentioned in the doc has not been published yet.
+
+# Contributing to Bicep public registry
+
+Currently, we only accept contributions from Microsoft employees. The guide was created to help teams within Microsoft with the development of Bicep public registry modules. External customers can propose a new module by opening an [issue]() in the [Azure/azure-quickstart-templates](https://github.com/Azure/azure-quickstart-templates/tree/master/modules) repository.
+
+## Prerequisite
+- Create a fork of the [Azure/azure-quickstart-templates](https://github.com/Azure/azure-quickstart-templates/tree/master/modules) repository and clone the fork to your local machine.
+<!-- TODO: Add link to Nuget once the tool is published -->
+- Install the [bicep-registry-module]() .NET tool by running:
+  - `dotnet tool install -g bicep-registry-module`
+
+## Creating a new module
+
+### Making a proposal for new module
+<!-- TODO: create an issue template in the quickstart repo -->
+Before creating a new module, you must fill out this [issue template](https://github.com/Azure/azure-quickstart-templates/issues/new) to make a proposal. Once the proposal is approved, proceed with the following steps.
```

Comment:
I'd emphasize that PRs without an associated approval will be rejected.

Reply:
Good point. Added a note.